README.md

```diff
@@ -26,5 +26,5 @@ Any questions let us know!
00: give your vote)</a></p></td><td align="center"><p><a href="https://github.com/vinzvinci98">Vincent Villafuerte</a></p><img src="https://user-images.githubusercontent.com/73097560/103608126-91b05900-4f55-11eb-84af-d30f20d9332c.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/215">(:100: give your vote)</a></p></td><td align="center"><p><a href="https://github.com/manzumrahman">Md. Manzum Rahman</a></p><img src="https://user-images.githubusercontent.com/63909405/103669582-baab0b00-4fa2-11eb-9e79-d21a269096a1.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/220">(:100: give your vote)</a></p></td></tr><tr><td align="center"><p><a href="https://github.com/TawfikYasser">Tawfik Yasser</a></p><img src="https://user-images.githubusercontent.com/54971231/103836002-6bdb9f00-5090-11eb-9501-dc21ccbf3607.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/227">(:100: give your vote)</a></p></td><td align="center"><p><a href="https://github.com/viraldevpb">Prathamesh Borse</a></p><img src="https://user-images.githubusercontent.com/66899360/104146755-cb6bde80-53f1-11eb-84bd-dbc87d80152e.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/232">(:100: give your vote)</a></p></td><td align="center"><p><a href="https://github.com/Samridhi-98">Samriddhi</a></p><img src="https://user-images.githubusercontent.com/54466041/104162252-40530e80-541a-11eb-967d-6b6f4050371c.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/235">(:100: give your vote)</a></p></td></tr><tr><td align="center"><p><a href="https://github.com/ritik307">Ritik Rawal</a></p><img src="https://user-images.githubusercontent.com/22374829/104178614-a8acea80-5430-11eb-8ee8-1a7bb89248b2.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/237">(:100: give your vote)</a></p></td><td align="center"><p><a href="https://github.com/filippo-fonseca">Filippo Fonseca</a></p><img src="https://user-images.githubusercontent.com/48065878/104632919-feb5a400-5663-11eb-8dd0-b05cfd377158.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/244">(:100: give your vote)</a></p></td><td align="center"><p><a href="https://github.com/paulsonjpaul">Paulson J Paul</a></p><img src="https://user-images.githubusercontent.com/73777251/104731775-360a7a80-5762-11eb-8a86-b50d98b58b01.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/247">(:100: give your vote)</a></p></td></tr><tr><td align="center"><p><a href="https://github.com/Olanetsoft">Idris Olubisi</a></p><img src="https://user-images.githubusercontent.com/45847909/104912483-9da40e00-598c-11eb-8881-971faeaecd9d.png" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/254">(:100: give your vote)</a></p></td><td></td><td></td></tr></table>
```

Comment:
The action will create this Tables for you. Please revert this change 😄

Reply:
I seen that the action ran on your fork. I've made a PR to fix this issue.

Reply:
Thank you @AllanRegush